api.planx.uk/gis/classifiedRoads.js

```diff
@@ -0,0 +1,60 @@
+import "isomorphic-fetch";
+
+const PASSPORT_FN = "roads.classified";
+
+export const classifiedRoadsSearch = async (
+  req,
+  res,
+  next
+) => {
+  try {
+    if (!req.query.geom)
+      return next({ status: 401, message: "Missing required query param `?geom=`" });
+
+    // Create an OGC XML filter parameter value which will select the road features (lines)
+    //   that intersect with the buffered site boundary (polygon) coordinates
+    //   ref https://labs.os.uk/public/os-data-hub-examples/os-features-api/wfs-example-intersects#maplibre-gl-js
+    const xml = `
+      <ogc:Filter">
```

Comment:
```suggestion
      <ogc:Filter>
```

Typo?

Reply:
:bulb: :pray: thank you :sweat_smile: you'd really think I'd have a better xml IDE extension by now for spotting these